gwctl/pkg/printer/backends.go

```diff
@@ -34,9 +34,20 @@ type BackendsPrinter struct {
 	Clock clock.Clock
 }
 
-func (bp *BackendsPrinter) Print(resourceModel *resourcediscovery.ResourceModel) {
+func (bp *BackendsPrinter) GetPrintableNodes(resourceModel *resourcediscovery.ResourceModel) []NodeResource {
+	return NodeResources(maps.Values(resourceModel.GatewayClasses))
+}
+
+func (bp *BackendsPrinter) PrintTable(resourceModel *resourcediscovery.ResourceModel, wide bool) {
+	var columnNames []string
+	if wide {
+		columnNames = []string{"NAMESPACE", "NAME", "TYPE", "REFERRED BY ROUTES", "AGE", "POLICIES"}
+	} else {
+		columnNames = []string{"NAMESPACE", "NAME", "TYPE", "REFERRED BY ROUTES", "AGE"}
```

Comment:
I know this is a bit different from what's documented but I've come to realize that we should avoid any parameter from being displayed which needs additional calculations. 

Calculating things like "REFERRED BY ROUTES" and "POLICIES" will take additional time (which becomes significant when there's lot's of resources). Hence it's preferrable to only include them in the `-o wide` format and not as a default.

tl;dr: Can you please move `REFERRED BY ROUTES` to wide format as well?

(In case you are wondering, we'll separately work on not calculating the additional values by default int the resourceModel -- as of now, that distinction doesn't exist`

Reply:
Of course, done!